--- ./pytorch-2.2.1/cmake/Modules/FindBLAS.cmake	2024-08-01 20:00:30.574422000 +0300
+++ cce_patches/FindBLAS.cmake	2024-10-02 23:25:28.656397000 +0300
@@ -105,6 +105,14 @@
   ENDIF(MKL_FOUND)
 endif()
 
+# AOCL Libsci?
+if((NOT BLAS_LIBRARIES))
+  SET(BLAS_INFO "AOCL")
+  SET(BLAS_LIBRARIES "blis") 
+  SET(BLAS_INCLUDE_DIR "/sw/ex111genoa/amd/aocl-aocc-4.2.0/4.2.0/aocc/include")
+  SET(BLAS_VERSION $ENV{AOCC_VERSION})
+  SET(BLAS_FOUND 1)
+endif()
 #BLIS?
 if((NOT BLAS_LIBRARIES)
     AND ((NOT WITH_BLAS) OR (WITH_BLAS STREQUAL "blis")))
